Source/relay/TourGuide/Quests/Level 11 - Hidden City.ash

```diff
@@ -355,7 +355,12 @@ void QLevel11HiddenCityGenerateTasks(ChecklistEntry [int] task_entries, Checklis
             } else if (hospital_progress == 0) {
                 generateHiddenAreaUnlockForShrine(subentry.entries,$location[an overgrown shrine (Southwest)]);
             } else {
-                boolean [item] outfitPieces = $items[bloodied surgical dungarees,surgical mask,head mirror,half-size scalpel].makeConstantItemArrayMutable();
+                boolean [item] outfitPieces = $items[bloodied surgical dungarees,surgical mask,head mirror].makeConstantItemArrayMutable();
+
+                # TODO maybe support it differently, the scalpel will still drop in that case
```

Comment:
i'm OK with this -- you're right, it's not a perfect fit but the logic is close enough